CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.18.2 / 2020-06-09
+
+* [BUGFIX] TSDB: Fixed incorrect fanout series merge causing incorrect query results in certain cases. #7361
```

Comment:
That is obscure for end users.

Suggestion:

TSDB: Fix corruption in query results when using remote read

Reply:
This is not even true, it has nothing to do to remote read.

Reply:
In fact remote read was not affected

Reply:
In practice it is with remote read that we merge series in the Prometheus server. Is there another case?

Reply:
Happy to change, what is not obscure but also true?

Reply:
I used the words "when using remote read" not "with remote read". The two bug reports from users mentioned remote read issues.

Reply:
sorry I misinterpret you, you are right actually from Prometheus user standpoint that's the only time it can be triggered :facepalm:

Reply:
>  when using remote read

Means for me when you call `/api/v1/read` or Prometheus that's why I was confused, will rephrase